<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spielfeld</class>
 <widget class="QWidget" name="Spielfeld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spielfeld</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_1"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>251</width>
     <height>220</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pbSchlachtschiff">
      <property name="text">
       <string>Schlachtschiff</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbKreuzer">
      <property name="text">
       <string>Kreuzer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbZerstoerer">
      <property name="text">
       <string>Zerstörer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbUboot">
      <property name="text">
       <string>U-Boot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ZufallsButton">
      <property name="text">
       <string>Zufällige Positionen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbDisconnect">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>84</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Trennen</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>7</y>
     <width>791</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelwin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblose">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>picture</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>gridLayoutWidget_2</zorder>
  <zorder>Start</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>pbDisconnect</zorder>
  <zorder>labelwin</zorder>
  <zorder>lblose</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
